import/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/backupuploader/BackupUploadServiceTest.java

```diff
@@ -179,17 +185,71 @@ public void startUploading_noMoreTry_BackupException() throws BackupException {
   public void startUploading_lastAttempt_noNewSchedule_BackupException() throws IOException, DicomWebException {
     spyBackupState = new BackupState(UNIQUE_FILE_NAME, ATTEMPTS_AMOUNT_ONE);
 
-    doThrow(new DicomWebException("Reason2")).doNothing().when(webClientMock).stowRs(any(InputStream.class));
-    when(delayCalculatorMock.getExponentialDelayMillis(anyInt())).thenReturn(0L);
+    doThrow(new DicomWebException("Reason2", HttpStatus.INTERNAL_SERVER_ERROR_500, HttpStatusCodes.STATUS_CODE_SERVER_ERROR))
```

Comment:
The third argument to DicomWebException should be Status.ProcessingFailure since it's a DIMSE code not HTTP